src/cli.ts

```diff
@@ -9,9 +9,9 @@ import * as util from "util";
 import { keyBy, flatten, chain } from "lodash";
 import { IMethodMapping } from "./router";
 import { resolve } from "path";
+import { version } from "../package.json"; // tslint:disable-line
```

Comment:
I think this only works using the require syntax or dynamic import

Reply:
when you publish to npm, package.json is in `./`, but this file is `./build/src/cli`

Reply:
I agree I think dynamic import would be the way to go here perhaps as well 🤔

Reply:
Interesting, I switch it back.

Reply:
I'll give you 100 pre-chain-ico BelfordTokens if you also make a test for the cli...

:trollface: